src/js/uswds-init.js

```diff
@@ -1,35 +1,18 @@
-const docElem = window.document.documentElement;
-const loadingClass = "usa-js-loading";
-
-function addLoadingClass() {
-  docElem.className += ` ${loadingClass}`;
-}
-
-function revertClass() {
-  docElem.className = docElem.className.replace(loadingClass, "");
-}
-
-// Based on https://www.filamentgroup.com/lab/enhancing-optimistically.html
-if ("querySelector" in window.document && "addEventListener" in window) {
-  addLoadingClass();
+(() => {
+  document.documentElement.classList.add("usa-js-loading");
+  function revertClass() {
+    document.documentElement.classList.remove("usa-js-loading");
+  }
```

Comment:
A few of the changes aren't strictly "best practices", but are implemented to take advantage of how gzip works to [optimize repeated strings](https://en.wikipedia.org/wiki/Deflate#Duplicate_string_elimination).

This alternative might be more conventional, but would increase size ever-so-slightly (+4 bytes gzipped):

```suggestion
  const loadingClass = "usa-js-loading";
  document.documentElement.classList.add(loadingClass);
  function revertClass() {
    document.documentElement.classList.remove(loadingClass);
  }
```

Reply:
> I'd lean towards the more convential style, but no strong opinions.

While it's outside the scope of this pull request, one thing I notice is that if ever the project migrates from Uglify to [Terser](https://github.com/terser/terser), Terser seems to compress the modified version to inline those strings anyways, so we'd get the best of both worlds: Smallest built file size, while keeping best practices in the source.

In the meantime, the few extra bytes aren't make or break, so I'll just apply the revision 👍